app/config/frontendAppConfig.scala

```diff
@@ -53,6 +53,8 @@ trait AppConfig extends ServicesConfig {
   val features: Features
   val emailVerificationBaseUrl: String
   val vatSubscriptionHost: String
+  val manageVatSubscriptionServiceUrl: String
+  val manageVatSubscriptionServicePath: String
```

Comment:
The tests aren't compiling because you haven't updated MockAppConfig to override these new values.